### PR TITLE
[Components] v7_darwin #12908

### PR DESCRIPTION
**Pull request by lcaresia**
## Summary by CodeRabbit

- **New Features**
	- Introduced action components for adding, creating, and updating datasets, enhancing user interaction with the application.
	- Added priority options for dataset management, allowing users to sort and prioritize their work effectively.

- **Bug Fixes**
	- Resolved various issues to improve the stability and responsiveness of the dataset actions.

- **Documentation**
	- Updated application documentation to reflect new features and usage instructions.

- **Chores**
	- Updated package dependencies to ensure compatibility and improved functionality.

### CODE REVIEW

File: components/v7_darwin/actions/add-instructions/add-instructions.mjs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import app from "../../v7_darwin.app.mjs";
+
+export default {
+  key: "v7_darwin-add-instructions",
+  name: "Add Instructions",
+  description: "Add annotator instructions. [See the documentation](https://docs.v7labs.com/reference/adding-instructions)",
+  version: "0.0.4",
```
Comment (luancazarine) [inline components/v7_darwin/actions/add-instructions/add-instructions.mjs:7]:
```suggestion
  version: "0.0.1",
```

**Review by luancazarine — CHANGES_REQUESTED** [review]
Hi @lcaresia, I just added a minor suggestion.

**Review by GTFalcao — APPROVED** [review]
LGTM!

**Pull request merged**